src/csharp/Microsoft.ML.OnnxRuntimeGenAI.csproj

```diff
@@ -0,0 +1,55 @@
+<Project Sdk="MSBuild.Sdk.Extras/3.0.22">
+  <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <Platforms>x64</Platforms>
+    <LangVersion>latest</LangVersion>
```

Comment:
LangVersion must be default so it is correctly set by the IDE according to the TargetFramework. This way it will not allow you to use language features that are not supported.

Reply:
Addressed this.